resources/qml/Sidebar.qml

```diff
@@ -33,6 +35,22 @@ Rectangle
         tooltip.hide();
     }
 
+    function strPadLeft(string, pad, length) {
```

Comment:
Quick! Let's remove this method and break all of nodejs!

(https://developers.slashdot.org/story/16/03/23/0652204/how-one-dev-broke-node-and-thousands-of-projects-in-11-lines-of-javascript for those that do not get it)